site/gdocs/pages/Homepage.scss

```diff
@@ -96,6 +96,13 @@
 }
 
 .homepage-social-ribbon__social-list {
+    grid-template-columns: 1fr; // fallback for browsers that don't support `max()` below.
+
+    // A bunch of magic numbers!
+    // The 40% ensures that auto-fit creates at most 2 columns. We can't use the more sensible 50%, because grid gaps need to be accounted for.
+    // The 120px ensures that a column is at least 120px wide, i.e. we only get 2 columns if the container is at least 240px (+gap) wide.
+    grid-template-columns: repeat(auto-fit, minmax(max(40%, 120px), 1fr));
```

Comment:
Damn, this took me a while to understand too 😅 

We have a `--grid-gap` var to make this slightly less magic, and you can have some element-scoped other variables to maybe? make it a bit more readable.
```css
--half-width: calc(50% - var(--grid-gap) / 2);
--min-column-width: max(var(--half-width), 120px);
grid-template-columns: repeat(
    auto-fit,
    minmax(var(--min-column-width), 1fr)
);
```